### ISSUE TITLE
MarkPriceUpdate is not array

### ISSUE THREAD

**Issue by ayan-dhara** [tiagosiebler/binance#217]
calling `wsClient.subscribeMarkPrice("BTCUSDT", "usdm", 1000)` return the data

```
{
  eventType: 'markPriceUpdate',
  eventTime: 1657338530003,
  symbol: 'BTCUSDT',
  markPrice: 21612.82096439,
  settlePriceEstimate: 21553.26911183,
  indexPrice: 21617.14783803,
  fundingRate: 0.0001,
  nextFundingTime: 1657353600000,
  wsMarket: 'usdm',
  wsKey: 'usdm_markPrice_btcusdt_'
}
```
but here
https://github.com/tiagosiebler/binance/blob/abf7741ab6c363fe1bc5e33ae0bbddd1519d05c6/src/util/typeGuards.ts#L21
it is checked to be array and converted `data` to `WsMessageMarkPriceUpdateEventFormatted[]`

**Comment by ayan-dhara** [comment]
may be you can add another function `isWsFormattedMarkPriceUpdateArray`

**Comment by tiagosiebler** [comment]
you're right, thanks for pointing this out - I'll get this fixed

**Comment by tiagosiebler** [comment]
@ayan-dhara 

Fixed in v2.0.34, there's now a method called `isWsFormattedMarkPriceUpdateEvent()`. The old `isWsFormattedMarkPriceUpdate()` method is deprecated (but remains for a bit), in favour of `isWsFormattedMarkPriceUpdateArray()`

**Comment by ayan-dhara** [comment]
nice, and thanks !!
